### PR TITLE
ID012 - Responding to friend requests

### PR DESCRIPTION
**Pull request by NikolasPasichnik**
To use it: 
1. Create two accounts and fill in your profile info on both 
2. Go on "Search Friend" and press on "Request Friend" to request the other person 
image
3. Go on the other account and press on "View Friend Requests" and press accept/decline to add the person or to decline their request. 
image
Note: I doesn't seem to refresh on my end, but you can see the request disappearing when you change tabs

### CODE REVIEW

File: web/src/components/Friendship/FriendRequests.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+    console.log("id: " + friendID);
+
+    //delete event with specified id
+    api
+      .deleteFriendRequest(await getAccessTokenSilently(), friendID)
+      .then(() => {
+        console.log("Deleting Friend Request");
+        getAllFriendRequests();
+      });
```
Comment (samperezh) [inline web/src/components/Friendship/FriendRequests.jsx:92]: For page to reload automatically after accepted invitation, add:  window.location.reload();

File: web/src/components/Friendship/FriendRequests.jsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+  async function declineFriendRequest(id) {
+    console.log("Declining Request");
+    console.log(id);
+
+    //delete event with specified id
+    api.deleteFriendRequest(await getAccessTokenSilently(), id).then(() => {
+      console.log("Deleting Friend Request");
+      getAllFriendRequests();
+    });
```
Comment (samperezh) [inline web/src/components/Friendship/FriendRequests.jsx:70]: For page to reload automatically after declining invitation, add:  window.location.reload();

**Review by samperezh — APPROVED** [review]
Could potentially remove some extra console.log() that are not necessary anymore to debug the code in order to keep the code clean if you have time. Check comments for fix so that page updates automatically after accepting/declining a request. Otherwise, LGTM!!

**Comment by NikolasPasichnik** [comment]
Request Friend works and Resolved Conflicts

**Pull request merged**